<?xml version="1.0"?>
<launch>
   <node pkg="path_follower" type="path_follower_node" name="path_follower" output="screen" respawn="true">
    <param name="use_obstacle_map" value="false" />
    <param name="use_vfh" value="false" />
    
    <param name="controller" value="omnidrive_vv" />
    <param name="cmd_vel" value="/robot_0/cmd_vel" />
    
    <param name="min_velocity" value="0.35" />
    <param name="max_velocity" value="0.9" />
    <param name="collision_box_width" value="0.4" />
    <param name="collision_box_crit_length" value="0.6" />
    <param name="collision_box_min_length" value="1.0" />
    <param name="collision_box_max_length" value="1.5" />
    <param name="collision_box_velocity_factor" value="1.0" />
    <param name="collision_box_velocity_saturation" value="1.0" />

    <param name="k" value="1.5" />
    <param name="kp" value="0.4" />
    <param name="kd" value="0.2" />
  </node>
  
</launch> 
